docs/conf.py

```diff
@@ -118,14 +118,24 @@
             "name": "Donate to NumFocus",
         },
     ],
-    "github_url": "https://github.com/pydata/pydata-sphinx-theme",
-    "twitter_url": "https://twitter.com/PyData",
+    # "github_url": "https://github.com/pydata/pydata-sphinx-theme",
+    # "twitter_url": "https://twitter.com/PyData",
```

Comment:
cruft:

```suggestion
```

Reply:
it's not cruft I wanted to leave them here as an example (as we support these parameters as well)